packages/pcd/zk-eddsa-frog-pcd/test/ZKEdDSAFrogPCD.spec.ts

```diff
@@ -47,7 +47,7 @@ const WATERMARK = BigInt(6);
 describe("ZKEdDSAFrogPCD should work", function () {
   this.timeout(1000 * 30);
 
-  let pcd: ZKEdDSAFrogPCD;
+  let pcd1: ZKEdDSAFrogPCD;
```

Comment:
pcd1 as a name leaves a little to be desired

Reply:
It's the first PCD to be made, isn't it obvious?  😜
Actually I copied it from another test which had multiple shared PCDs between test cases.  I can call this sharedPCD or something to point out that it's intended to be shared between tests.